OpenUpSA/wazimap-ng-ui
# adding values on the barchart

## Description
Added the values of the barchart at the end of the bars so they are visible without having to hover. When user filters or switches between the display types(percentage / value) the appended values change too.

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/156

## How to test it locally
* Run the project
* Expand the rich data panel
* Check if the values of the bars are also appended at the end of the bars.

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/100542290-fb07c280-3259-11eb-9751-50dafa39bfdc.png)
![image](https://user-images.githubusercontent.com/53019884/100542297-09ee7500-325a-11eb-83ac-8e8b14fac067.png)


## Changelog

### Added

### Updated
* horizontal-bar-chart.js to create `text` elements
* chart.js to modify the marging

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
